src/components/SearchBox.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Card, Input } from "@hackclub/design-system";
+
+const Base = Card.withComponent(Input).extend.attrs({
+    borderRadius: props => props.theme.pill,
+    boxShadowSize: "sm",
+    f: 5,
+    mx: "auto",
+    mt: 4,
+    px: 4
+})`
```

Comment:
again, without seeing the site, might look better with a `transition` (and, you got it, `will-change`)